core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -407,8 +410,13 @@ export class LoggedOutUser extends BaseUser {
     await this.page.$eval(weCannotContentId, element =>
       element.getElementsByTagName('a')[0].click()
     );
+    await this.page.waitForNavigation({waitUntil: ['load', 'networkidle2']});
```

Comment:
To use this waitForNavigation correctly, make sure to call it while the button is being clicked. 
Example:
`await Promise.all([this.page.waitForNavigation(), this.clickOn(button)]);`

It avoids the navigation timeout error when this.page.waitForNavigation() started after the navigation is fully completed.

Reply:
Done.